querydsl-sql/src/test/java/com/querydsl/sql/SubqueriesBase.java

```diff
@@ -15,7 +29,11 @@
 import com.google.common.collect.ImmutableList;
 import com.querydsl.core.testutil.ExcludeIn;
 import com.querydsl.core.types.SubQueryExpression;
-import com.querydsl.core.types.dsl.*;
+import com.querydsl.core.types.dsl.Expressions;
+import com.querydsl.core.types.dsl.NumberPath;
+import com.querydsl.core.types.dsl.Param;
+import com.querydsl.core.types.dsl.PathBuilder;
```

Comment:
We collapse our imports when we use a package a lot.
So we add wildcards when there are 5 imports from the same package and 3 for static imports.
Could you also apply that to your other PR? That adds less noise and allows us to review it better,